app/controllers/api/v0/search_controller.rb

```diff
@@ -1,8 +1,8 @@
 class Api::V0::SearchController < Api::V0::BaseController
 
   EXCEPTIONS_422 = [
-    Search::BookVersions::SearchStrategies::UnknownSearchStrategy,
-    Search::BookVersions::SearchStrategies::IncompatibleStrategies
+    Books::SearchStrategies::S1::UnknownSearchStrategy,
```

Comment:
These should not be scoped under `S1`.  They are common across strategies of which `S1` is one.